layouts/pages/search.ejs

```diff
@@ -17,7 +17,7 @@
               <% if (res.folder) { %>
                 <%- template('search.results.label.folder', res.folder.path, res.folder.prettyName || 'Home')%>&nbsp;
               <% } %>
-              <% if (res.lastModifyingUser) { %>
+              <% if (res.lastModifyingUser && res.lastModifyingUser.displayName) { %>
                 <%- template('search.results.label.person', res.lastModifyingUser.displayName) %>
               <% } %>
               <%= res.lastUpdated %>
```

Comment:
Looks like this line may be broken too — I think the field name is `modifiedTime`, not `lastUpdated`. 

It might be nice to update the template function to take `res` and return a message that reads "Last edited by `res.lastModifyingUser.displayName` on `res.lastUpdated`".